tests/run_test.sh

```diff
@@ -306,6 +303,7 @@ NODES=$(( TASKS / TPN ))
 if (( NODES * TPN < TASKS )); then
   NODES=$(( NODES + 1 ))
 fi
+TASKS=$(( NODES * TPN ))
```

Comment:
Why is TASKS recomputed again? This will mpiexec model with more MPI tasks than required by decomposition and write grid comp parameters. Is this required by new esmf threading?